src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const resultInverse = {};
+  const seenValues = new Set();
```

Comment:
it's unnecessary

Reply:
https://github.com/mate-academy/js_inverse_robot/pull/2630/files#r1294579343
didn't fix it